From: Stained Glass OS <ke7oxh@gmail.com>
Subject: [PATCH] msvcp60: Export time_get::date_order and do_date_order.

msvcp60's time_get<char> and time_get<unsigned short>::date_order and
do_date_order, the msvcp90 sources' functions (msvcp60's names return
int where later ones return time_base::dateorder).
---
 dlls/msvcp60/msvcp60.spec | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/dlls/msvcp60/msvcp60.spec b/dlls/msvcp60/msvcp60.spec
index ee72800..b800a10 100644
--- a/dlls/msvcp60/msvcp60.spec
+++ b/dlls/msvcp60/msvcp60.spec
@@ -3343,11 +3343,11 @@
 @ thiscall -arch=i386 ?data@?$basic_string@GU?$char_traits@G@std@@V?$allocator@G@2@@std@@QBEPBGXZ(ptr) MSVCP_basic_string_wchar_c_str
 @ cdecl -arch=win64 ?data@?$basic_string@GU?$char_traits@G@std@@V?$allocator@G@2@@std@@QEBAPEBGXZ(ptr) MSVCP_basic_string_wchar_c_str
 @ stub -arch=arm ?date_order@?$time_get@DV?$istreambuf_iterator@DU?$char_traits@D@std@@@std@@@std@@QBAHXZ
-@ stub -arch=i386 ?date_order@?$time_get@DV?$istreambuf_iterator@DU?$char_traits@D@std@@@std@@@std@@QBEHXZ
-@ stub -arch=win64 ?date_order@?$time_get@DV?$istreambuf_iterator@DU?$char_traits@D@std@@@std@@@std@@QEBAHXZ
+@ thiscall -arch=i386 ?date_order@?$time_get@DV?$istreambuf_iterator@DU?$char_traits@D@std@@@std@@@std@@QBEHXZ(ptr) time_get_char_date_order
+@ cdecl -arch=win64 ?date_order@?$time_get@DV?$istreambuf_iterator@DU?$char_traits@D@std@@@std@@@std@@QEBAHXZ(ptr) time_get_char_date_order
 @ stub -arch=arm ?date_order@?$time_get@GV?$istreambuf_iterator@GU?$char_traits@G@std@@@std@@@std@@QBAHXZ
-@ stub -arch=i386 ?date_order@?$time_get@GV?$istreambuf_iterator@GU?$char_traits@G@std@@@std@@@std@@QBEHXZ
-@ stub -arch=win64 ?date_order@?$time_get@GV?$istreambuf_iterator@GU?$char_traits@G@std@@@std@@@std@@QEBAHXZ
+@ thiscall -arch=i386 ?date_order@?$time_get@GV?$istreambuf_iterator@GU?$char_traits@G@std@@@std@@@std@@QBEHXZ(ptr) time_get_wchar_date_order
+@ cdecl -arch=win64 ?date_order@?$time_get@GV?$istreambuf_iterator@GU?$char_traits@G@std@@@std@@@std@@QEBAHXZ(ptr) time_get_wchar_date_order
 @ cdecl -arch=arm ?decimal_point@?$_Mpunct@D@std@@QBADXZ(ptr) mpunct_char_decimal_point
 @ thiscall -arch=i386 ?decimal_point@?$_Mpunct@D@std@@QBEDXZ(ptr) mpunct_char_decimal_point
 @ cdecl -arch=win64 ?decimal_point@?$_Mpunct@D@std@@QEBADXZ(ptr) mpunct_char_decimal_point
@@ -3398,11 +3398,11 @@
 @ thiscall -arch=i386 ?do_curr_symbol@?$_Mpunct@G@std@@MBE?AV?$basic_string@GU?$char_traits@G@std@@V?$allocator@G@2@@2@XZ(ptr ptr) mpunct_short_do_curr_symbol
 @ cdecl -arch=win64 ?do_curr_symbol@?$_Mpunct@G@std@@MEBA?AV?$basic_string@GU?$char_traits@G@std@@V?$allocator@G@2@@2@XZ(ptr ptr) mpunct_short_do_curr_symbol
 @ stub -arch=arm ?do_date_order@?$time_get@DV?$istreambuf_iterator@DU?$char_traits@D@std@@@std@@@std@@MBAHXZ
-@ stub -arch=i386 ?do_date_order@?$time_get@DV?$istreambuf_iterator@DU?$char_traits@D@std@@@std@@@std@@MBEHXZ
-@ stub -arch=win64 ?do_date_order@?$time_get@DV?$istreambuf_iterator@DU?$char_traits@D@std@@@std@@@std@@MEBAHXZ
+@ thiscall -arch=i386 ?do_date_order@?$time_get@DV?$istreambuf_iterator@DU?$char_traits@D@std@@@std@@@std@@MBEHXZ(ptr) time_get_char_do_date_order
+@ cdecl -arch=win64 ?do_date_order@?$time_get@DV?$istreambuf_iterator@DU?$char_traits@D@std@@@std@@@std@@MEBAHXZ(ptr) time_get_char_do_date_order
 @ stub -arch=arm ?do_date_order@?$time_get@GV?$istreambuf_iterator@GU?$char_traits@G@std@@@std@@@std@@MBAHXZ
-@ stub -arch=i386 ?do_date_order@?$time_get@GV?$istreambuf_iterator@GU?$char_traits@G@std@@@std@@@std@@MBEHXZ
-@ stub -arch=win64 ?do_date_order@?$time_get@GV?$istreambuf_iterator@GU?$char_traits@G@std@@@std@@@std@@MEBAHXZ
+@ thiscall -arch=i386 ?do_date_order@?$time_get@GV?$istreambuf_iterator@GU?$char_traits@G@std@@@std@@@std@@MBEHXZ(ptr) time_get_wchar_do_date_order
+@ cdecl -arch=win64 ?do_date_order@?$time_get@GV?$istreambuf_iterator@GU?$char_traits@G@std@@@std@@@std@@MEBAHXZ(ptr) time_get_wchar_do_date_order
 @ cdecl -arch=arm ?do_decimal_point@?$_Mpunct@D@std@@MBADXZ(ptr) mpunct_char_do_decimal_point
 @ thiscall -arch=i386 ?do_decimal_point@?$_Mpunct@D@std@@MBEDXZ(ptr) mpunct_char_do_decimal_point
 @ cdecl -arch=win64 ?do_decimal_point@?$_Mpunct@D@std@@MEBADXZ(ptr) mpunct_char_do_decimal_point
